extensions/wikia/Search/classes/Test/FandomSearchTest.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Wikia\Search\Test;
+
+use ReflectionMethod;
+
+class FandomSearchTest extends BaseTest {
+	const CLASS_NAME = 'Wikia\Search\FandomSearch';
+
+	public function testBuildUrl() {
+		$method = new ReflectionMethod( static::CLASS_NAME, 'buildUrl' );
```

Comment:
It's generally better to avoid testing private methods, especially one with such simple logic as this, as they are implementation details. If this method is reusable outside the class, it should be public or in a separate helper, otherwise what do you think of instead testing the public interface (in this case `getStories`) which will also allow you to verify that the result of buildUrl (and `pareResult`) is correct when mocking the dependency on `Http::request`?